### ISSUE TITLE
[Filter Assistant] - I'm not sure the audience to know how to comment

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-eval-tools#93]
Just commentary that this seems to be written for people who understand the Web and technology in general (e.g., know what "browser plugin" means or care about a11y down to details such as automated vs. manual testing). 

Most folks I encounter simply want to know "how do I comply with WCAG" or even "how do we make sure we're accessible"? Thus, they may come to this assistant simply to find, say, free tools that can show them what's missing in their website and what to fix (without having to learn WCAG or about accessibility or even about Web technology). 

I don't necessarily think it's a bad thing to focus on the actual ways of accessibility; it's not this form's purpose to educate people about Web technologies. Instead, this can provide a means of people understanding that accessibility is not a simple checklist action you take when you're thinking about it (and never again) and know accessibility is an entire professional field that takes some education to properly implement. 

Nevertheless, perhaps there's an "I'm lost" or "I'm new" link that takes users to an Intro to A11y page or has a pop-up paragraph that explains more?? Just concerned we're definitely going to leave non-technical people lost (e.g., Program Managers tasked with accessibility at their company, lawyers investigating lawsuits filed, non-profit volunteers trying to understand how to be inclusive in their digital presence, etc.).

**Comment by VeraLange** [comment]
Hey Michele,

We've tried to accommodate this issue by including a link to the "Selecting Web Accessibility Evaluation Tools"-page (to be updated), and the filter assistant, which gives additional guidance especially to newer tool users. 

We hope those both form good starting points for users. To prevent scope creep of this resource, we have decided against more changes to the page that would accommodate more inexperienced tool users even more.
